lib/src/__tests__/DateRangePicker.test.tsx

```diff
@@ -80,11 +26,9 @@ describe('DateRangePicker', () => {
       />
     );
 
-    component
-      .find('input')
-      .at(1)
-      .simulate('focus');
-
-    expect(component.find('div[role="tooltip"]').length).toBe(1);
+    fireEvent.focus(getByMuiTest('start-input'));
+    const el = await screen.findAllByRole('tooltip')
+    screen.debug(el[1]);
```

Comment:
@eps1lon It could be interesting to have an eslint rule that fails if a debug is left in the tests.